.gitattributes

```diff
@@ -0,0 +1,16 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
+
+# Explicitly declare text files you want to always be normalized and converted
+# to native line endings on checkout.
+*.c text
+*.h text
+*.py text eol=lf
+
+# Declare files that will always have CRLF line endings on checkout.
+*.sln text eol=crlf
+
+# Denote all files that are truly binary and should not be modified.
+*.png binary
+*.jpg binary
+*.pyc binary
```

Comment:
I believe this is not needed either, because *.pyc files are not even tracked (they are ignored because of the setting in the .gitignore file that prevent files inside `__pycache__` folders to even be acknowledged).

In other words, git ignores `__pycache__` folders and their contents completely, so there isn't even any need to worry about their settings. Please, let me know if I'm missing something since I'm not that nearly as knowledgeable in Python as people might assume I am.